vnext/JSI/Shared/ChakraRuntime.h

```diff
@@ -328,7 +329,8 @@ class ChakraRuntime : public facebook::jsi::Runtime {
   bool evaluateSerializedScript(
       const facebook::jsi::Buffer &scriptBuffer,
       const facebook::jsi::Buffer &serializedScriptBuffer,
-      const std::string &sourceURL);
+      const std::string &sourceURL,
+      JsValueRef *result);
```

Comment:
>result [](start = 18, length = 6)

what is the lifetime of this pointer? Shouldn't it be shared_ptr or something like that?

Reply:
It is to return the value. Its lifetime is provided by the caller. I will add a comment.

---
In reply to: [460195714](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r460195714) [](ancestors = 460195714)

Reply:
(nit) Would it be possible to switch it to "&"? or we want to support nullptr case?

---
In reply to: [460582169](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r460582169) [](ancestors = 460582169,460195714)